rts/Sim/Path/QTPFS/Node.h

```diff
@@ -146,13 +147,23 @@ namespace QTPFS {
 			bool& needSplit
 		);
 
+		bool UpdateExitOnly(
+			NodeLayer& nl,
+			bool& needSplit
+		);
+
 		static unsigned int MIN_SIZE_X;
 		static unsigned int MIN_SIZE_Z;
 
 	public:
 		static unsigned int MAX_DEPTH;
 
 	private:
+		static constexpr unsigned int NODE_INDEX_MASK = 0x000fffff;
```

Comment:
looks like 12 mask bits on the left and 20 index bits on the right, why this proportion? do we expect more mask bits?